tests/test_base.py

```diff
@@ -79,6 +79,8 @@ def test_quote(self):
             ["foo/bar ha/barz", "foo/bar%20ha/barz"],
             ["example b.txt", "example%20b.txt"],
             ["exampleㄓ.txt", "example%E3%84%93.txt"],
+            ["test/colon:tmp", "test/colon%3Atmp"],
+            ["test/forward\\slash", "test/forward/slash"],
```

Comment:
This looks wrong? It means that **both** `"test/forward\\slash"` and `"test/forward/slash"`would get transformed to `"test/forward/slash"` ??

Reply:
before quoting,  path_snippet = path_snippet.replace("\\", "/") did it.

Reply:
Yeah, but surely they should still be different _after_ quoting, otherwise an attempt to access the quoted-version of `"test/forward\\slash"` will actually give you `"test/forward/slash"` which isn't what you want?

```bash
$ echo short > /tmp/file\\foo:bar
$ mkdir /tmp/file
$ echo "really really long" > /tmp/file/foo:bar
$ ls -l /tmp/file\\foo:bar
-rw-rw-r-- 1 qpc qpc 6 Aug  8 22:37 /tmp/file\foo:bar
$ ls -l /tmp/file/foo:bar
-rw-rw-r-- 1 qpc qpc 19 Aug  8 22:37 /tmp/file/foo:bar
```

Reply:
`["test/forward\\slash", "test/forward%5Cslash"]` will make you happy

Reply:
...except I always thought that `\` was the backward-slash? :wink:

Reply:
see the updated commit and I will put it as resolved.